SmartDeviceLink/public/SDLChoiceSet.m

```diff
@@ -144,6 +121,40 @@ - (void)setHelpList:(nullable NSArray<SDLVRHelpItem *> *)helpList {
     }
 }
 
+#pragma mark - Helpers
+
+/**
+ Check for duplicate choices and voiceCommands
+
+ @param choices The choices you will be adding
+ @return Boolean that indicates whether choices and voice commands are unique or not
+ */
+-(BOOL)sdl_choiceCellsAreUnique:(NSArray<SDLChoiceCell *> *)choices {
```

Comment:
If the developer creates a set of `SDLChoiceCell` with duplicate `text` set and different `secondaryArtwork`s but the module does not support `secondaryArtwork`, then you end up with a bunch of rows with the exact same name:
![Duplicate cell names](https://user-images.githubusercontent.com/19915981/109161898-94ede700-7745-11eb-868c-bf809737f181.png)

Reply:
After investigating this, this is also possible just by creating the same cell except for different VR commands. I think that we should just add a documentation warning that if uniquing features are not displayed on screen then the same information will be displayed across multiple cells. I added this to the initializers for `SDLChoiceSet`:

```
WARNING: If you display multiple cells with the same `text` with the only uniquing property between cells being different `vrCommands` or a feature that is not displayed on the head unit (e.g. if the head unit doesn't display `secondaryArtwork` and that's the only uniquing property between two cells) then the cells may appear to be the same to the user in `Manual` mode. This only applies to RPC connections >= 7.1.0.

WARNING: On < 7.1.0 connections, the `text` cell will be automatically modified among cells that have the same `text` when they are preloaded, so they will always appear differently on-screen when they are displayed. `cell.uniqueText` will be created by appending ` (2)`, ` (3)`, etc.
```